hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-infra/src/main/resources/start-component.sh

```diff
@@ -94,6 +94,12 @@ function hdfs_original {
 extraClasspathDir="$(pwd)/additionalClasspath/"
 mkdir -p "${extraClasspathDir}"
 
+junitClassPath="$(find $hadoopHome -name "junit*.jar" | head -n 1)"
```

Comment:
shellcheck:24: note: Double quote to prevent globbing and word splitting. [SC2086]